src/pages/Guides/Guides.tsx

```diff
@@ -62,12 +62,12 @@ function Guides(props: GuidesProps) {
       </div>
       <div className={classes.scroll}>
         {articles.map((key: any) => (
-          <Link
-            style={{ textDecoration: 'none' }}
-            to={`${props.match.url}/Instructions/${key}`}
-          >
-            <ResourceCard resource={resources[key]} />
-          </Link>
+          <ResourceCard
```

Comment:
Same comment as above — at this point the code pretty much looks the same regardless of whether we're rendering articles or videos, so we should be able to just write this once and do a map over all the types we'd like to render (articles with the header "Instructions", and videos with the header "Videos).

Not a crucial change though since I know we're short on time, but we can keep this in mind for the future if we want to clean up our code for maintainability!